mozilla_django_oidc/views.py

```diff
@@ -114,15 +114,15 @@ def redirect_url(self):
 
     def dispatch(self, request, *args, **kwargs):
         """Log out the user."""
-
-        if request.user.is_authenticated():
+        # Check if a method exists to build the URL to log out the user
+        # from the OP.
+        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
```

Comment:
I am not sure whether we should log out the user from the OP's session in the case where the user in not logged in in the app. Probably, the expected behavior would be to do that only when the user is logged in and the relevant setting is present. @peterbe @johngian @willkg thoughts?